docs/trouble_shooting.md

```diff
@@ -27,3 +27,37 @@ This guide (based on minikube and others should be similar) explains general inf
    ```
    # kubectl --kubeconfig minikube.kubeconfig log clusterapi-controller-0 -n ibmcloud-provider-system
    ```
+
+## Cannot create bootstrap cluster if you are using kind
+   
+1. Check if kind works well.
```

Comment:
1) delete `1.` here.
2) Run `doctoc` https://github.com/thlorenz/doctoc to update the toc.
3) Squash your commits to one. Ask @xunpan or @morvencao if you do not know how to squash.